src/workers/postUpdated.ts

```diff
@@ -0,0 +1,68 @@
+import { messageToJson, Worker } from './worker';
+import { ArticlePost, SharePost } from '../entity';
+
+interface Data extends ArticlePost {
+  updated_at: Date;
+}
+
+const worker: Worker = {
+  subscription: 'api.post-updated',
+  handler: async (message, con, logger): Promise<void> => {
```

Comment:
This handler misses a use case for crawler and community picks where we don't prematurely generate an id that Yggdrasil can utilize. Am I right?

Reply:
Ah good point, not sure it's high impact for now, but we can add it.
So far we have that retro use case only for later down the line.

Let me know what you think.

Reply:
@idoshamun We need to think this trough, for community picks we need to change it regardless so the post will be created same as with squads now.
For crawler I agree it's an exception, but what rules will we set since it's not "updating" then but creating a completely new one.

Reply:
Please confirm with Viktor that we can really tackle it later. If so, that's what will do. Ask him on #engineering please